src/main/resources/db/migration/R__Comments.sql

```diff
@@ -474,4 +474,6 @@ COMMENT ON TABLE accelerator.submission_statuses IS '(Enum) Statuses of submissi
 
 COMMENT ON TABLE accelerator.submissions IS 'Information about the current states of the information supplied by specific projects in response to deliverables.';
 
+COMMENT ON TABLE accelerator.vote_options IS '(Enum) Available vote options.';
+COMMENT ON TABLE accelerator.project_votes IS 'Vote options (nullable) selected by voters.';
```

Comment:
This comment should go up above line 468 to keep the tables in alphabetical order within the `accelerator` schema.